packages/core/src/render3/node_manipulation.ts

```diff
@@ -517,8 +517,8 @@ function getRenderParent(tView: TView, tNode: TNode, currentView: LView): REleme
       // Since the projection would then move it to its final destination. Note that we can't
       // make this assumption when using the Shadow DOM, because the native projection placeholders
       // (<content> or <slot>) have to be in place as elements are being inserted.
-      if (encapsulation !== ViewEncapsulation.ShadowDom &&
-          encapsulation !== ViewEncapsulation.Native) {
+      // TODO(FW-2290): remove the `encapsulation !== 1` fallback logic in v12.
+      if (encapsulation !== ViewEncapsulation.ShadowDom && encapsulation !== 1) {
```

Comment:
Why note use?

```suggestion
      if (encapsulation === ViewEncapsulation.None || encapsulation === ViewEncapsulation.Emulated) {
```

Then you don't need to worry about removing the `1` later.

Reply:
I think this would also be preferred since we can't be sure that adding a new `ViewEncapsulation` type in the future would also allow this early escape.